db/db_impl_open.cc

```diff
@@ -412,6 +412,12 @@ Status DBImpl::Recover(
     default_cf_handle_ = new ColumnFamilyHandleImpl(
         versions_->GetColumnFamilySet()->GetDefault(), this, &mutex_);
     default_cf_internal_stats_ = default_cf_handle_->cfd()->internal_stats();
+    if (mutable_db_options_.stats_persist_period_sec != 0) {
```

Comment:
Shall we always open the persist stats CF, since you provide an `PersistentStatsColumnFamily` API?

Reply:
+1. If the user does not want to use it, the column family can still be there with very low overhead.

Reply:
@yiwu-arbug @riversand963 we can't do that. We need an option to turn it on to keep the DB being able to be downgraded to an older version that can't consume this CF.

Reply:
I see @siying . Just to confirm. If we start with a new version of RocksDB enabling persistent stats column family, and downgrade to an old version that does not support this feature, the persistent stats column family will be treated as an ordinary column family by the old version?

Reply:
@riversand963 but since users aren't aware of stats CF (at least in their older version), they will fail to open this DB. This can create a maintenance challenge for them.